scalalib/src/Dep.scala

```diff
@@ -74,6 +74,12 @@ case class Dep(dep: coursier.Dependency, cross: CrossVersion, force: Boolean) {
       case _ =>
         this
     }
+  def withPlatformed(platformed: Boolean): Dep =
```

Comment:
Here I'm not sure if it's better this or
```suggestion
  def withPlatformed(value: Boolean): Dep =
```